src/providers/packet/mock_tests.rs

```diff
@@ -34,3 +34,42 @@ fn test_boot_checkin() {
     mock.assert();
     r.unwrap();
 }
+
+#[test]
+fn test_packet_attributes() {
+    let metadata = r#"{
+        "id": "test-id",
+        "hostname": "test-hostname",
+        "iqn": "test-iqn",
+        "plan": "test-plan",
+        "facility": "test-facility",
+        "tags": [],
+        "ssh_keys": [],
+        "network": {
+            "interfaces": [],
+            "addresses": [],
+            "bonding": { "mode": 0 }
+        },
+        "phone_home_url": "test-url"
+    }"#;
+
+    let hostname = "test-hostname";
+    let phone_home_url = "test-url";
+    let plan = "test-plan";
+
+    let attributes = maplit::hashmap! {
+        format!("PACKET_HOSTNAME") => String::from(hostname),
```

Comment:
You can use `xyz.to_string()` on both sides of the arrow (here and below).

Reply:
:+1: 
Updating!